src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CreateSomaticPanelOfNormals.java

```diff
@@ -46,45 +51,27 @@
  *
  * <h4>Step 1. Run Mutect2 in tumor-only mode for each normal sample.</h4>
  * <pre>
- * gatk Mutect2 \
- *   -R reference.fa \
- *   -I normal1.bam \
- *   -tumor normal1_sample_name \
- *   -O normal1_for_pon.vcf.gz
+ * gatk Mutect2 -R reference.fasta -I normal1.bam -O normal1.vcf.gz
  * </pre>
  *
- * <h4>Step 2. Create a file ending with .args or .list extension with the paths to the VCFs from step 1, one per line.</h4>
- * <p>This approach is optional. Other extensions will error the run. </p>
+ * <h4>Step 2. Create a GenomicsDB from the normal Mutect2 calls.</h4>
+ * Note that GenomicsDBImport is currently (as of February 2019) inefficient when processing multiple intervals.  Therefore,
+ * we recommend creating a separate GenomicsDB and panel of normals for each chromosome, then merging the panels.  This is
+ * done automatically in mutect2_pon.wdl.  Here we show the process for chromosome 20.
  *
- * <pre>
- *     normal1_for_pon.vcf.gz
- *     normal2_for_pon.vcf.gz
- *     normal3_for_pon.vcf.gz
- * </pre>
+ *  <pre>
+ *    gatk GenomicsDBImport -R reference.fasta -L 20\
+ *       --genomicsdb-workspace-path pon_db \
+ *       -V normal1.vcf.gz \
+ *       -V normal2.vcf.gz \
+ *       -V normal3.vcf.gz
```

Comment:
you could presumably also use a sample_map if you have a lot of samples like Lee

Reply:
I would hope that anyone doing this on a large scale would be running via the WDL and wouldn't have to worry about an unwieldy command.  But it can't hurt to mention, I suppose.